docs/getting-started.md

```diff
@@ -85,15 +85,16 @@ defmodule MyApp.Router do
       info: %{
         version: "1.0",
         title: "My App"
-      }
+      },
+      basePath: "/api"
     }
   end
 end
 ```
 
 The `version` and `title` are mandatory fields. By default the `version` will be `0.0.1`
 and the `title` will be `<enter your title>` if you do not provide `swagger_info/0`
-function.
+function. `basePath` is optional but may need to be specified if your API routes do not reside at the root location `/`.
```

Comment:
Thanks for the PR.
Would you mind moving this into the paragraph below which is intended to cover the non-mandatory fields that are commonly used.